daily-challenges/2024-09/18/khanh5923_18.spec.js

```diff
@@ -0,0 +1,41 @@
+function calculateAge(yearBirth){
+    const yearCurrent = new Date().getFullYear();
+    return yearBirth > yearCurrent ? console.log(`Nam sinh khong hop le`) : console.log(`Tuoi cua ban la: ${yearCurrent-yearBirth}`)
+}
+calculateAge(2003)
+
+// ---------------------------------------------------------
+
+import {test, expect} from '@playwright/test'
+test('bai4_18', async ({ page }) => {
+    await page.goto(`https://material.playwrightvn.com/`)
+    await page.click(`//a[contains(text(),'Bài học 3: Todo page')]`)
+    const testData = {
+        text: "Xin chào, đây là bài thực hành ngày 18 tháng 9",
+        update: "Xin chào, đây là bài thực hành ngày 18 tháng 9 - phiên bản đã chỉnh sửa"
+    }
+    await page.locator(`//input[@id='new-task']`).fill(testData.text)
+    await page.locator(`//button[@id='add-task']`).click()
+    const onlyAdd = page.locator(`//ul[@id="task-list"]//li`)
```

Comment:
Đặt tên solo quá. Nên đặt tên có chủ đích và ý nghĩa.
```suggestion
    const taskListLocator = page.locator(`//ul[@id="task-list"]//li`)
```

Reply:
Dạ để em khắc phục